gobcore/model/gobmodel.json

```diff
@@ -3787,6 +3787,157 @@
             "description": "De naam van het land waar het adres zich bevindt."
           }
         }
+      },
+      "vestigingen": {
+        "version": "0.1",
+        "abbreviation": "VES",
+        "description": "Een Vestiging is gebouw of een complex van gebouwen waar duurzame uitoefening van activiteiten van een Onderneming of Rechtspersoon plaatsvindt.",
+        "entity_id": "vestigingsnummer",
+        "attributes": {
+          "vestigingsnummer": {
+            "type": "GOB.String",
+            "description": "Betreft het identificerende gegeven voor de vestiging."
+          },
+          "tijdstip_registratie": {
+            "type": "GOB.DateTime",
+            "description": "De datum waarop de vestiging bij de KvK in het handelsregister is geregistreerd."
+          },
+          "datum_aanvang": {
+            "type": "GOB.Date",
+            "description": "De datum waarbij de vestiging is ontstaan."
+          },
+          "datum_einde": {
+            "type": "GOB.Date",
+            "description": "De datum waarbij de vestiging is beëindigd."
+          },
+          "datum_voortzetting": {
+            "type": "GOB.Date",
+            "description": "Datum voortzetting van de vestiging."
+          },
+          "is_commerciele_vestiging": {
+            "type": "GOB.Boolean",
+            "description": "Geeft aan of dit een commerciële vestigign betreft. Als false, dan een niet-commerciële vestiging."
```

Comment:
Typo "vestigign'

Reply:
Nice catch! Fixed this and added this to the other open GOB-Core PR